docker-compose.yml

```diff
@@ -20,7 +20,8 @@ services:
                 },
                 "pubsweet-server": {
                     "baseUrl": "${PUBSWEET_BASEURL}",
-                    "secret": "somesecret"
+                    "secret": "somesecret",
+                    "hostname": "${PUBSWEET_HOSTNAME}"
```

Comment:
cross-checked that this comes from https://github.com/elifesciences/elife-xpub-deployment/blob/develop/.env#L6